src/config/index.ts

```diff
@@ -1,4 +1,4 @@
-const PROD_HOST_URL = 'https://web.ente.io';
+const PROD_HOST_URL = 'https://save-key-on-desktop.bada-frame.pages.dev/';
```

Comment:
Was this intentional?

Reply:
this was for testing.... had to revert back... thanks for pointing out